wgsl/index.bs

```diff
@@ -11388,6 +11416,10 @@ Note: The result is not mathematically meaningful when `abs(e)` &ge; 1.
     <td>Returns either `min(max(e, low), high)`, or the median of
     the three values `e`, `low`, `high`.
     [=Component-wise=] when `T` is a vector.
+
+    If `low` is greater than `high`, then:
+    * It is a [=shader-creation error=] if `low` and `high` are [=const-expressions=].
+    * It is a [=pipeline-creation error=] if `low` and `high` are [=override-expressions=].
```

Comment:
The nit-picker in me wants to ask: What if `low` is a `const-expression` and `high` is an `override-expression` or vice-versa.

Reply:
Right.  There's already overlap:  every const-expression is an override-expression.
In the previous PR we talked about not having to explicitly call out the overlaps because of the general rule at https://gpuweb.github.io/gpuweb/wgsl/#processing-errors 

> Each requirement will be checked at the earliest opportunity. 

So if some set of conditions satisfies both shader-creation and pipeline-creation error conditions, then the shader-creation error is the actual required result.